packages/odata-service-inquirer/package.json

```diff
@@ -31,8 +31,10 @@
     ],
     "dependencies": {
         "@sap/cf-tools": "3.2.2",
+        "@sap-ux/annotation-converter": "0.9.10",
```

Comment:
These should be `workspace:*` ?

Reply:
No, actually these are in a different repo.